core/src/main/scala/libretto/CoreLib.scala

```diff
@@ -1909,15 +1909,15 @@ class CoreLib[DSL <: CoreDSL](val dsl: DSL) { lib =>
       f.zoomCo(Functor[[x] =>> B1 |*| x])
   }
 
-  implicit class FocusedOnPlusContra[A, F[_], B1, B2](f: FocusedContra[F, B1 |+| B2]) {
+  extension [A, F[_], B1, B2](f: FocusedContra[F, B1 |+| B2]) {
     def left: FocusedContra[λ[x => F[x |+| B2]], B1] =
       f.zoomCo(|+|.left[B2])
 
     def right: FocusedContra[λ[x => F[B1 |+| x]], B2] =
       f.zoomCo(|+|.right[B1])
   }
 
-  implicit class FocusedOnChoiceContra[A, F[_], B1, B2](f: FocusedContra[F, B1 |&| B2]) {
+  extension [A, F[_], B1, B2](f: FocusedContra[F, B1 |&| B2]) {
```

Comment:
All cases need comments, imho,. Same as before.

Reply:
Extends the focus to the left/right side of the (currently focused) consumer choice.